DigitalLearningSolutions.Data/Services/UserService.cs

```diff
@@ -133,6 +149,12 @@ public bool TryUpdateUserAccountDetails
             {
                 var delegateIds = verifiedDelegateUsers.Select(d => d.Id).ToArray();
                 userDataService.UpdateDelegateUsers(firstName, surname, email, profileImage, delegateIds);
+
+                if (verifiedDelegateUsers.Any(u => u.Id == delegateId))
+                {
+                    userDataService.UpdateDelegateUserCentrePrompts(delegateId!.Value, jobGroupId!.Value, answer1, answer2, answer3,
```

Comment:
To avoid the jobGroupId!.Value, we could pass in an optional CentrePromptsAnswers? object, which will only be populated if the user is delegate, but will have the correct strong types for the fields.